src/i18n/en-US/index.ts

```diff
@@ -210,6 +210,8 @@ export default {
     dappStakingEvm: 'dApp Staking is available on EVM',
     onChainData: 'On-Chain Data',
     stakingTvl: 'Staking TVL',
+    cantClaimWihtoutError:
+      'Looks like you are not able to claim with automatic re-stake because you have not claimed in a while. Please turn off the Auto Re-Stake feature to be able to claim. After you claim rewards you can turn on re-stake again. UI team is working to fix this issue.',
```

Comment:
Please change:
You cannot claim with automatic re-stake because it's been a while since you claimed your rewards. Please turn off the Auto Re-Stake feature to be able to claim. After you claim rewards you can turn on re-stake again. The UI team is working to fix this issue.